omnigibson/robots/manipulation_robot.py

```diff
@@ -382,11 +382,11 @@ def release_grasp_immediately(self):
         for arm in self.arm_names:
             if self._ag_obj_in_hand[arm] is not None:
                 self._release_grasp(arm=arm)
+                self._ag_release_counter[arm] = int(np.ceil(m.RELEASE_WINDOW / og.sim.get_rendering_dt()))
```

Comment:
Merging og-develop into action_primitives first should make this diff disappear (Josiah already merged this).